crowdsourcing/serializers/user.py

```diff
@@ -135,15 +137,18 @@ def create(self, **kwargs):
         user_profile = models.UserProfile()
         user_profile.user = user
         user_profile.save()
-        worker = models.Worker()
-        worker.profile = user_profile
-        worker.alias = username
-        worker.save()
-
-        requester = models.Requester()
-        requester.profile = user_profile
-        requester.alias = username
-        requester.save()
+
+        if 'is_worker' not in self.validated_data or self.validated_data['is_worker']:
```

Comment:
no actually right now the user is created as a worker and requester by default (as before). is that what we want? if we dont, then you are right it should be **in** not **not in**.